<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefreshPkg</class>
 <widget class="QMainWindow" name="RefreshPkg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RefreshPkg</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;设备调试软件 &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="v_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;版本号v1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;SN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>661</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;MAC地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;IP地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;设备型号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;设备类型&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>设备搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectAll">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unselectAll">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>全不选</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ip_list_widget">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>160</y>
      <width>671</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ROM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>APK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathRom">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathApk">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectorRom">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectorApk">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始刷机</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditLog">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>721</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;配置状态&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>36</width>
      <height>8</height>
     </rect>
    </property>
    <property name="text">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;功能选择&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFace">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>569</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>人脸识别框</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxScan">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>605</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>扫码开门</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPrivacy">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>640</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>隐私保护</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMask">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>674</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>口罩提醒</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMove">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>574</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>移动侦测</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxTemperature">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>605</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>温度显示</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxWakeup">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>640</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>智能感应唤醒</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAccess">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>owlook平台</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>未知</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelAccess">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>556</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>接入方式</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSvrAddr">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>590</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSvrAddr">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>593</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemperature">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>618</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>测温模块</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTemperature">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>620</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>测温模块</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelDoor">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>650</y>
      <width>36</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>开门方式</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDoor">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>650</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>刷卡或刷脸</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>刷卡</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>刷脸</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelLanguage">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>682</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>语言选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLanguage">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>680</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>中文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonImport">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>560</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入配置文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExport">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>612</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成配置文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
